scripts/main.js

```diff
@@ -35,6 +35,9 @@ module.exports = (robot) => {
     robot.hear(/badger/i, (res) => {
         res.send("Badgers? BADGERS? WE DON'T NEED NO STINKIN BADGERS")
     })
+    robot.hear(/ourbot help/i, (res) => {
+        res.send("Hi, I'm your helpful chatops bot! Please, see README for the usage https://github.com/datopian/ourbot#how-to-use-it")
```

Comment:
@rufuspollock Yes, it was created yesterday, i modified as you suggested. Trying to write unittest. Btw, you can check now writing `bot` or `/bot` to get a response